_includes/module-loading/bundling-content-css-files.md

```diff
@@ -0,0 +1,45 @@
+## Community
+
+Default
+: ```
+./skins/content/default/content.css
```

Comment:
do we need any additional context to what i need to do with these css file paths?

Reply:
we show them in the walkthroughs, as well as on the reference page where this include is used: http://docs-v5-hotfix-doc-376.staging.tiny.cloud/docs/advanced/usage-with-module-loaders/reference/content-css/